pandas/_libs/groupby_helper.pxi.in

```diff
@@ -439,6 +466,9 @@ def group_max(groupby_t[:, :] out,
         # Note: evaluated at compile-time
         maxx[:] = -_int64_max
         nan_val = NPY_NAT
+    elif groupby_t is uint64_t:
+        maxx[:] = 0
+        nan_val = 0  # We better not need this!  TODO: Can we use e.g. NULL?
```

Comment:
Can you clarify what this is?

Reply:
We have to define a value to treat as NA, and it has to be of the appropriate dtype.  But for uint64 there _is_ no such value, so defining it here is just as a placeholder.

I'll clarify the comment in the code.